src/approvals-service.ts

```diff
@@ -284,3 +285,83 @@ export async function getFormApprovalFlows(
     }
   }
 }
+
+export async function getFormSubmissionAdministrationApprovals(
+  formsAppId: number,
+  formId?: number,
+  externalId?: string,
+  submissionid?: string,
+  submittedAfterDateTime?: string,
+  submittedBeforeDateTime?: string,
+  limit?: number,
+  offset?: number,
```

Comment:
Can we change this to an object, instead of individual arguments? JS does not have the helpers like C# to skip an argument. Can make it really easy to pass the wrong value for the wrong argument.